src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/pdl/dto/PdlPerson.kt

```diff
@@ -237,6 +237,13 @@ enum class AdressebeskyttelseGradering {
     UGRADERT,
 }
 
+fun AdressebeskyttelseGradering.tilDiskresjonskode(): String? = when (this) {
+    AdressebeskyttelseGradering.STRENGT_FORTROLIG -> "SPSF" // Kode 6
+    AdressebeskyttelseGradering.STRENGT_FORTROLIG_UTLAND -> "SPSF" // Kode 19
```

Comment:
Det er meningen at disse skal være like?
Og hva er kode 19? 🤓

Reply:
strengt fortrolig utland? 
Kanskje skrive "strengt fortrolig" osv i stedet for kode 6/7, evt. begge deler siden det er dette som faktisk bruker nå og ikke krever at folk er klar over hva det het før

Reply:
Kode 19 er det egentleg kun Bidrag som har eit forhold til, mens resten av systemene skal tolke utland som ein kode 6.

Reply:
Tenker at strengt fortrolig ligger i navnet på enumen. Kode 6/719 er bare til hjelp 😎

Reply:
Hehe ja, så ikke det da jeg kommenterte. Enig blir dobbelt 🤓